Doc/whatsnew/3.14.rst

```diff
@@ -771,6 +771,92 @@ asyncio
 * Removed implicit creation of event loop by :func:`asyncio.get_event_loop`.
   It now raises a :exc:`RuntimeError` if there is no current event loop.
   (Contributed by Kumar Aditya in :gh:`126353`.)
+  There's a few patterns that use :func:`asyncio.get_event_loop`, most
+  of them can be replaced with :func:`asyncio.run`.
+
+  If you're running an async function, simply use :func:`asyncio.run`,
+  before::
+
+    async def amain():
+        ...
+
+    loop = asyncio.get_event_loop()
+    try:
+       loop.run_until_complete(amain())
+    finally:
+       loop.close()
+
+  after::
+
+    async def amain():
+        ...
+
+    asyncio.run(amain())
+
+  If you need to start something, eg, a server listening on a socket
+  and then run forever, use :func:`asyncio.run` and an
```

Comment:
I don't think this server example is useful to add, I'm fine with the Runner one though

Reply:
I feel that it really belongs to an asyncio recipes section rather than whatnew page, this pattern orthogonal to the change listed here. What do you think?

Reply:
I really think it's useful here, perhaps it can be added to recipes later?

Eg for users of 3.15+